api/src/main/java/com/epam/esm/handler/ResourceNotFoundedExceptionHandler.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.esm.handler;
+
+import com.epam.esm.exception.ResourceNotFoundedException;
+import com.epam.esm.entity.ResponseExceptionEntity;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+/**
+ * The ResourceNotFoundedException handler.
+ *
+ * @author Yauheni Tstiov
+ */
+@ControllerAdvice
+public class ResourceNotFoundedExceptionHandler {
+    private static final HttpStatus status = HttpStatus.NOT_FOUND;
+    private static final int ERROR_CODE = 40401;
```

Comment:
According to the task, the error code should be calculated using the following parts:

- response status (e.g. 404)
- entity code (e.g. `01` - for certificates, `02` - for tags, `09` - unknown)
And this true for all kind of errors.

Reply:
Done. Create ErrorCodeCounter for count error code and add in custom runtime exception codeExceptionCause field.